server/tests/integration/automation-scheduler.test.js

```diff
@@ -262,57 +258,56 @@ describe('Automation controller', () => {
 
     it('should cancel a job', async () => {
         await dbCleaner(async () => {
-            await scheduleCollectionJobByMutation();
+            const { scheduleCollectionJob: job } =
+                await scheduleCollectionJobByMutation();
             const {
                 collectionJob: { cancelCollectionJob: cancelledCollectionJob }
-            } = await cancelCollectionJobByMutation();
+            } = await cancelCollectionJobByMutation(job.id);
 
-            expect(cancelledCollectionJob).toEqual({
-                id: jobId,
-                status: 'CANCELLED'
-            });
+            expect(cancelledCollectionJob.status).toEqual('CANCELLED');
             const { collectionJob: storedCollectionJob } =
-                await getTestCollectionJob();
-            expect(storedCollectionJob.id).toEqual(jobId);
+                await getTestCollectionJob(job.id);
             expect(storedCollectionJob.status).toEqual('CANCELLED');
         });
     });
 
     it('should gracefully reject request to cancel a job that does not exist', async () => {
         expect.assertions(1); // Make sure an assertion is made
-        await expect(cancelCollectionJobByMutation()).rejects.toThrow(
+        await expect(cancelCollectionJobByMutation(2)).rejects.toThrow(
             'Could not find collection job with id 2'
         );
     });
 
     it('should restart a job', async () => {
         await dbCleaner(async () => {
-            await scheduleCollectionJobByMutation();
+            const { scheduleCollectionJob: job } =
+                await scheduleCollectionJobByMutation();
             const { restartCollectionJob: collectionJob } =
-                await restartCollectionJobByMutation();
+                await restartCollectionJobByMutation(job.id);
             expect(collectionJob).not.toBe(undefined);
             expect(collectionJob).toEqual({
-                id: jobId,
+                id: job.id,
                 status: 'QUEUED'
             });
             const { collectionJob: storedCollectionJob } =
-                await getTestCollectionJob();
-            expect(storedCollectionJob.id).toEqual(jobId);
+                await getTestCollectionJob(job.id);
+            expect(storedCollectionJob.id).toEqual(job.id);
             expect(storedCollectionJob.status).toEqual('QUEUED');
         });
     });
 
     it('should gracefully reject restarting a job that does not exist', async () => {
         const { restartCollectionJob: res } =
-            await restartCollectionJobByMutation();
+            await restartCollectionJobByMutation('2');
```

Comment:
All the others seem to be taking ints instead of strings now, feel like this might've been missed.

Reply:
Oh good point! It doesn't matter for how the code functions because it's being put into a GraphQL query but I swapped it for an int here for clarity. [26d851a](https://github.com/w3c/aria-at-app/pull/925/commits/26d851a993af7b1fc30922a22efd3c76a93895b9)